<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RomAssessWindow</class>
 <widget class="QMainWindow" name="RomAssessWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PLUTO ROM Assessment</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Bahnschrift Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>PLUTO ROM Assessment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="hocGraph"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pbArom">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Assess AROM</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbProm">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Assess PROM</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textInstruction">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>729</width>
         <height>71</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>729</width>
         <height>71</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(240, 240, 240);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
